### DISCUSSION TITLE
Canvas multiple init error

### DISCUSSION THREAD

**Discussion by w934423231** [fabricjs/fabric.js#8144] [Q&A]
I made a project in vue2 using fabricjs 5.2.1. When switching between multiple canvases, there was a problem in selecting the elements. From the data point of view, it is possible to select, but the page display is wrong
![image](https://user-images.githubusercontent.com/38958128/184281225-90edd83b-1f35-4f16-b5b3-4ca8f2270e2c.png)
Normally, it should be like this
https://user-images.githubusercontent.com/38958128/184281510-d8381e0a-4d45-4cba-9270-5fb928427153.mp4
However, problems arise when multiple canvases are switched

**Comment by w934423231** [comment]
![image](https://user-images.githubusercontent.com/38958128/184282548-d8470a4e-91a6-4dc3-a2b1-95d1bca5581b.png)
From the display of data and mouse, it is possible to select and move, but the page is not displayed. I don't know why. Most of the logic is processed by fabricjs

**Comment by ShaMan123** [comment]
I suggest you stick to an **INFORMATIVE and CONCISE** title.
And look into the answers of your previous bugs because they all seem to originate from the same error #8126.
Please avoid spamming the repo - do your research.

**Reply by w934423231** [reply]
No, the two problems do not belong to the same problem. The reason for the last problem is that I know the canvas ID causes them to conflict. I'll be fine after I solve the problem. But I don't know where the problem is,
